feat(acc): partner space doc (#3230)
scaleway/docs-content: partners/partner-space/how-to/lock-client-organization.mdx

---
meta:
  title: How to lock a client's Organization
  description: This page explains how to lock a client's Organization from the Partner Space.
content:
  h1: How to lock a client's Organization
  paragraph: This page explains how to lock a client's Organization from the Partner Space.
tags: partner-space 
dates:
  validation: 2024-05-27
  posted: 2024-05-27
categories:
  - partner-space
---

> If you are confronted with a client who does not respect Scaleway’s Terms of Service, you can lock their Organisation. 

Reviewer: ```suggestion
If you are confronted with a client who does not respect Scaleway’s Terms of Service, you can lock their Organization. 
```

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)

> - Scaleway Partner status

Reviewer: ```suggestion
- Scaleway Partner status
- [Added a client](/partners/partner-space/how-to/add-client/)
```

  <Message type="important">

>      Locking an Organization prevents its owner to order or manage resources. .

Reviewer: ```suggestion
     Locking an Organization prevents its owner from ordering or managing resources.
```

    </Message>

> 1. Click the **See more** button (<Icon name="more" />) to the right of the client’s creation date. 

Reviewer: As before

Reviewer 2: ```suggestion
1. Click <Icon name="more"> to the right of the client’s creation date. 
```

> 1. Click the **See more** button (<Icon name="more" />) to the right of the client’s creation date. 

Reviewer 3: ```suggestion
1. Click <Icon name="more" /> to the right of the client’s creation date. 
```
